fedireads/outgoing.py

```diff
@@ -81,7 +81,7 @@ def handle_account_search(query):
 def handle_follow(user, to_follow):
     ''' someone local wants to follow someone '''
     activity = activitypub.get_follow_request(user, to_follow)
-    errors = broadcast(user, activity, [to_follow.inbox])
+    errors = broadcast.delay(user.id, activity, [to_follow.inbox])
```

Comment:
Rather than rewriting every caller would it be better to leave these calls as is and call the delayed broadcast from the broadcast function?